.github/workflows/continuous-integration-linux.yml

```diff
@@ -105,6 +126,7 @@ jobs:
             -DCMAKE_EXE_LINKER_FLAGS="${{ matrix.extra_linker_flags }}" \
             -DCMAKE_CXX_COMPILER=${{ matrix.cxx }} \
             -DCMAKE_CXX_COMPILER_LAUNCHER=ccache \
+            ${{ matrix.cmake_extra_opts }} \
```

Comment:
In some way `matrix.clang-tidy` is already the same `cmake_extra_opts`, but it felt too invasive to change it everywhere, it would be cleaner though.

Reply:
@masterleinad @dalg24 Should I rename `matrix.clang-tidy` to `matrix.cmake_extra_opts` to clean this up and reuse?